x-pack/plugins/security_solution/common/ecs/index.ts

```diff
@@ -74,5 +74,5 @@ export interface Ecs {
   Target?: Target;
   dll?: DllEcs;
   'kibana.alert.workflow_status'?: 'open' | 'acknowledged' | 'in-progress' | 'closed';
-  'kibana.alert.rule.parameters'?: { index: string[] };
+  'kibana.alert.rule.parameters'?: { index: string[]; data_view_id?: string };
```

Comment:
Are these meant to correspond to existing ECS fields? I see `dataViewId` exists [here](https://www.elastic.co/guide/en/security/8.3/timeline-object-schema.html) but that's different than this one I think?

Reply:
I'm not entirely sure.. I don't think they correspond directly to the ECS fields. I think these are the explicit parameters for the rule as they come over the wire. I can add a comment here to denote that.

Reply:
@dplumlee touched these in https://github.com/elastic/kibana/pull/122387, he should be able to confirm this.

Reply:
cc @marshallmain